nexus/db-queries/src/db/datastore/db_metadata.rs

```diff
@@ -499,8 +679,9 @@ mod test {
             .await;
 
         // Show that the datastores can be created concurrently.
-        let config =
-            SchemaConfig { schema_dir: config_dir.path().to_path_buf() };
+        let config = SchemaConfig {
+            schema_dir: config_dir.into_path().into_std_path_buf(),
```

Comment:
I think this is failing CI: https://docs.rs/camino-tempfile/1.0.2/camino_tempfile/struct.Utf8TempDir.html#method.into_path

> This consumes the Utf8TempDir without deleting directory on the filesystem, meaning that the directory will no longer be automatically deleted.

Reply:
Whoops! Thanks for pointing this out.